io.github.TheWisker.Cavasik.json

```
@@ -0,0 +1,83 @@
+{
+    "app-id" : "io.github.TheWisker.Cavasik.desktop",
```

there is never `.desktop` in the app-id.

```suggestion
    "app-id" : "io.github.TheWisker.Cavasik",
```

---

Thanks. Fixed it!